testlapack/matgen.go

```diff
@@ -12,6 +12,15 @@ import (
 	"github.com/gonum/blas/blas64"
 )
 
+const (
```

Comment:
`dlamchE` is used elsewhere than just `matgen.go`, so these should probably be in `general.go`.